search_gov_crawler/search_gov_spiders/pipelines.py

```diff
@@ -5,9 +5,8 @@
 
 import os
 from pathlib import Path
-
 from scrapy.exceptions import DropItem
-
+import fcntl
```

Comment:
if you are naming the files uniquely, do you still need the fnctl lock?

Reply:
We are still using multiple threads per spider.

Reply:
Are we? I really don't know... haha.  I removed it and still go the same total count while running two spiders.  Fine to leave in.

Reply:
Yeah, in the item pipeline specifically. -> https://docs.scrapy.org/en/latest/topics/settings.html#std:setting-CONCURRENT_REQUESTS_PER_IP:~:text=the%20startproject%20command.-,CONCURRENT_ITEMS,-%C2%B6

Reply:
I could change this back to the with lock that it was before if we don't want to import fnctl